package pruebinas;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;

public class TestConnection {

	public static void main(String[] args) throws SQLException {
		Connection con = null;
		String username = "root";
        String password = "Pedret@2";
		con = DriverManager.getConnection("jdbc:mysql://localhost:3306/", username, password);
        /* java.sql.Statement st = con.createStatement();
        ResultSet mar = st.executeQuery("SELECT * FROM table");
    Connection conn = null;

        conn = DriverManager.getConnection(
        		"jdbc:mysql://localhost:3306/Floristeria" +
                        "user=root&password=Pedret@2");
    
    System.out.println("Connected to database");
		
		    /*   try {
		            // The newInstance() call is a work around for some
		            // broken Java implementations

		            Class.forName("com.mysql.cj.jdbc.Driver");
		  
			} catch (Exception ex) {
			    // handle any errors
			    System.out.println("SQLException: " + ex.getMessage());
			    System.out.println("SQLException: " + ex.getCause());
			}*/


		}


	}

